name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/SantanderConsumerBankPDFExtractor.java

```diff
@@ -256,6 +256,31 @@ private void addAccountStatementTransaction()
                             t.setNote(TextUtil.replaceMultipleBlanks(v.get("note")).trim());
                         })
 
+                        .wrap(TransactionItem::new));
+
+        // 20.05.2023 20.05.2023 -1,00 33,01 Auszahlung auf ...
+        Block removalBlock = new Block(
+                        "^(?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) ([\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) (?<amount>[\\.,\\d-]+) ([\\.,\\d-]+) (Auszahlung) (?<note>.*)");
```

Comment:
- Improve regulare expression
- Remove obsolet mapping

```suggestion
                        "^[\\d]{2}\\.[\\d]{2}\\.[\\d]{4} [\\d]{2}\\.[\\d]{2}\\.[\\d]{4} \\-[\\.,\\d]+ [\\.,\\d]+ Auszahlung .*$");
```